application/src/main/java/com/github/pieter_groenendijk/model/DigitalProduct.java

```diff
@@ -0,0 +1,66 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "DigitalProduct")
+public class DigitalProduct extends Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long digitalProductId;
+
+    @Column(name = "title", nullable = false, length = 100)
+    private String accessStatus;
+
+    @Column(name = "releaseDate", nullable = true)
+    private Date releaseDate;
+
+    @Column(name = "classification", nullable = true)
+    private int classification;
```

Comment:
Wat zijn voorbeelden van een `classification`?

Ik zie dat het een `int` is, ik vemoed dus een soort enum structuur?

Reply:
Via Teams beschrijf je dat het gaat om een age classification. Wellicht kan het dan `ageClassification` of `minimumAge` heten.

Opzich lijkt het mij wel zo flexibel om hier gebruik te maken van een `int`. Het is de meest letterlijke domeinwaarde, dus daar kan je later nog alles mee. Als de kolom een goede naam heeft, voegt een enum naar mijn mening niet super veel toe.